contrib/release/start-release.sh

```diff
@@ -113,12 +113,12 @@ main() {
     else
       $DIR/prep-changelog.sh "$old_version" "$version"
     fi
-    git commit -a -s -m "Prepare for release $version"
 
     logecho "Next steps:"
-    logecho "* Check the new release commit with 'git show'"
+    logecho "* Check all changes and add to a new commit"
     logecho "  * If this is a prerelease, create a revert commit"
     logecho "* Push the PR to Github for review ('submit-release.sh')"
+    logecho "* Close https://github.com/cilium/cilium/projects/$old_proj"
```

Comment:
This was already done by the cilium/release script before we even run this script right?

Reply:
I think yes, I'll remove this line